src/TaxRate.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace PodPoint\I18n;
+
+use Mpociot\VatCalculator\VatCalculator;
+
+class TaxRate
+{
+    /**
+     * @var VatCalculator
+     */
+    private $vatCalculator;
+
+    public function __construct(VatCalculator $vatCalculator)
+    {
+        $this->vatCalculator = $vatCalculator;
+    }
+
+    /**
+     * Returns the tax rate for the given country code.
+     *
+     * @param string $countryCode
+     *
+     * @return float
+     */
+    public function get(string $countryCode): float
+    {
+        return (float) $this->vatCalculator->getTaxRateForCountry($countryCode);
```

Comment:
is there need for `(float)` ?
getTaxRateForCountry apparently returns float and even if, php return type annotation should get the job done anyway.

Reply:
this will apply for the other methods as well